templates/default/MODULE.bazel

```diff
@@ -0,0 +1,2 @@
+bazel_dep(name="rules_ll", version="20230218.0")
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
```

Comment:
I don't think that we need `bazel_skylib` here. We use it in the examples for the tests [here](https://github.com/eomii/rules_ll/blob/main/examples/BUILD.bazel), but the `rules_ll`-internal usages of the `bazel_skylib` import it implicitly.